.circleci/config.yml

```
@@ -1,150 +1,17 @@
-
-    - run:
-        # Project specific: Delete built-in kube-dns + CoreDNS in this project,
-        # since it cannot run side-by-side with one installed by the test.
-        name: Delete built-in core-dns
-        command: |
-          export KUBECONFIG=$(./e2ectl kubeconfig path)
-          ./kubectl delete service kube-dns -n kube-system
-          ./kubectl delete deployment coredns -n kube-system
-          ./kubectl delete configmap coredns -n kube-system
-          ./kubectl delete clusterrolebinding system:coredns
-          ./kubectl delete serviceaccount coredns -n kube-system
-          ./kubectl delete clusterrole system:coredns
-
-    - run:
-        name: Run test
-        command: |
-          docker run --rm --network host \
-            -v $(pwd)/${E2E_TEST_DIR}:/e2e \
-            -v $(pwd):/go/src/github.com/${CIRCLE_PROJECT_USERNAME}/${CIRCLE_PROJECT_REPONAME} \
-            -e E2E_KUBECONFIG=/e2e/kubeconfig \
-            -e CIRCLE_SHA1=${CIRCLE_SHA1} \
-            -w /go/src/github.com/${CIRCLE_PROJECT_USERNAME}/${CIRCLE_PROJECT_REPONAME} \
-            golang:1 go test -v -tags k8srequired ./${E2E_TEST_DIR}/
-
-    - run:
-        name: Export logs
-        command: |
-          ./e2ectl logs export --logs-dir=kind/logs
-        when: always
-
-    - store_artifacts:
-        path: kind/logs
-        destination: logs
-
-    - run:
-        name: Finish with cleanup, no matter if the test succeeded or not
-        command: |
-          ./e2ectl cluster delete
-          ./architect unpublish --channels=${CIRCLE_SHA1}-${E2E_TEST_NAME}
-        when: always
-
-version: 2
-jobs:
-  build:
-    machine: true
-    steps:
-    - checkout
-
-    - run:
-        name: Get architect binary
-        command: |
-          wget -q $(curl -sS -H "Authorization: token $RELEASE_TOKEN" https://api.github.com/repos/giantswarm/architect/releases/latest | grep browser_download_url | head -n 1 | cut -d '"' -f 4)
-          chmod +x ./architect
-          ./architect version
-
-    - run: ./architect build
-
-    - run:
-        name: Publish chart to CNR using a temporary channel for feature branch deploys
-        command: ./architect publish --pipeline=false --channels=wip-${CIRCLE_SHA1}
-
-    - persist_to_workspace:
-        root: .
-        paths:
-        - ./architect
-
-  e2eTestBasic:
-    environment:
-      E2E_TEST_DIR: "integration/test/basic"
-      E2E_TEST_NAME: "basic"
-    <<: *e2eTest
-
-  deploy:
-    machine: true
-    steps:
-    - checkout
-
-    - attach_workspace:
-        at: .
-
-    - run: ./architect deploy
-
-    - run:
-        name: Publish chart to beta channel on CNR
-        command: ./architect publish
-
-  publish_to_stable:
-    machine: true
-    steps:
-    - checkout
-
-    - attach_workspace:
-        at: .
-
-    - run: ./architect publish --stable
+version: 2.1
+orbs:
+  architect: giantswarm/architect@0.2.0
```

```suggestion
  architect: giantswarm/architect@0.4.2
```

See https://github.com/giantswarm/giantswarm/blob/master/processes/appcatalog.md#packaging-and-pushing-apps-into-an-app-catalog.